lib/rubocop/options.rb

```diff
@@ -416,6 +418,13 @@ def validate_display_only_failed_and_display_only_correctable
             format('--display-only-failed cannot be used together with other display options.')
     end
 
+    def validate_lsp_and_editor_mode
+      return if !@options.key?(:lsp) || !@options.key?(:editor_mode)
```

Comment:
It may be clearer to use `return unless options.key(:lsp) && @options.key(:editor_mode)`?

Reply:
Following De Morgan's laws, it's questionable whether `unless` is appropriate in this case. Anyway, the current condition will be maintained here.